components/chatbot/inputBox.js

```diff
@@ -132,13 +131,10 @@ function InputBoxComponent({
                     </span>
                   </div>
                   <button
-                    className={
-                      "transition-all duration-200 relative font-semibold rounded-md px-3 py-1.5 text-sm text-white ring-blue-600 active:ring-0 ring-0 hover:ring-0 outline-none hover:outline-none focus:outline-none border-0 h-full opacity-75" +
-                      (isSendChatLoading
-                        ? " opacity-40 text-white "
-                        : " bg-blue-600 text-white")
-                    }
-                    onClick={handleClick}
+                    className={classNames("transition-all duration-200 relative font-semibold rounded-md px-3 py-1.5 text-sm text-white ring-blue-600 active:ring-0 ring-0 hover:ring-0 outline-none hover:outline-none focus:outline-none border-0 h-full opacity-75", isSendChatLoading
```

Comment:
can use the  https://www.npmjs.com/package/classnames to handle dynamic class names